event_sourcery_esdb/__init__.py

```diff
@@ -19,5 +21,8 @@ def __init__(self, esdb: EventStoreDBClient) -> None:
         self._configure(storage_strategy=ESDBStorageStrategy(self._client))
 
     def with_outbox(self, filterer: OutboxFiltererStrategy = dummy_filterer) -> Self:
+        temporal_outbox = f"outbox-test-{uuid4().hex}"
```

Comment:
uhmm did you mean temporary? EDIT: just found out temporal can have similar meaning, nvm - if it's not a typo, let's leave it although it made me wonder at the first sight.

Reply:
maybe `tmp_` ? :D